sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-storage-blob/pom.xml

```diff
@@ -30,7 +30,7 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-storage-blob</artifactId>
-      <version>12.6.0</version>  <!-- {x-version-update;com.azure:azure-storage-blob;dependency} -->
+      <version>12.7.0-beta.1</version>  <!-- {x-version-update;com.azure:azure-storage-blob;dependency} -->
```

Comment:
revert?

Reply:
@gapra-msft this is actually correct since that is the beta version.

Reply:
Jim said keep this one.